GitUI/MessageBoxes.cs

```diff
@@ -68,27 +70,21 @@ public static void NotValidGitDirectory([CanBeNull] IWin32Window owner)
             => ShowError(owner, Instance._notValidGitDirectory.Text);
 
         public static void ShowGitConfigurationExceptionMessage(IWin32Window owner, GitConfigurationException exception)
-        {
-            MessageBox.Show(owner,
-                string.Format(ResourceManager.Strings.GeneralGitConfigExceptionMessage, exception.ConfigPath, Environment.NewLine, (exception.InnerException ?? exception).Message),
-                ResourceManager.Strings.GeneralGitConfigExceptionCaption,
-                MessageBoxButtons.OK, MessageBoxIcon.Warning);
-        }
+            => Show(owner,
+                    string.Format(ResourceManager.Strings.GeneralGitConfigExceptionMessage,
+                                  exception.ConfigPath, Environment.NewLine, (exception.InnerException ?? exception).Message),
+                    ResourceManager.Strings.GeneralGitConfigExceptionCaption,
+                    MessageBoxButtons.OK,
+                    MessageBoxIcon.Warning);
 
         public static bool UnresolvedMergeConflicts(IWin32Window owner)
```

Comment:
...linked to the above:
```suggestion
        public static DialogResult ConfirmResolveMergeConflicts(IWin32Window owner)
            => Show(owner, text, caption, MessageBoxButtons.YesNo, MessageBoxIcon.Question)
```

Reply:
This will make the interface of `MessageBoxes` depend more on that of `MessageBox`.
We might want to replace `MessageBox` with `TaskDialog` in the future.

Reply:
> We might want to replace `MessageBox` with `TaskDialog` in the future.

❤️ 
That's on my ever growing backlog. The .NET 5.0 TaskDialog is test-enabled, i.e. a dialog can be closed programmatically despite the static nature of invocation - the dialog can be access via a bound page.